[CI] EsqlSpecForceStoredLoadingIT class failing

**Build Scans:**
- [elasticsearch-intake #40949 / part3](https://gradle-enterprise.elastic.co/s/eomr7eq2lee2o)
- [elasticsearch-pull-request #152426 / part-3](https://gradle-enterprise.elastic.co/s/kugnkgefq7wzc)
- [elasticsearch-pull-request #152426 / java_rest_tests](https://gradle-enterprise.elastic.co/s/2cmjdn2hnn7rk)
- [elasticsearch-pull-request #152426 / part-3](https://gradle-enterprise.elastic.co/s/tpblsklw5rn6i)
- [elasticsearch-pull-request #152421 / part-3-fips-140-3](https://gradle-enterprise.elastic.co/s/scd4wijhr264q)
- [elasticsearch-pull-request #152253 / part-3](https://gradle-enterprise.elastic.co/s/fewuf7ynlxycc)
- [elasticsearch-pull-request #152329 / part-3](https://gradle-enterprise.elastic.co/s/hksxw2ycoqcf4)
- [elasticsearch-pull-request #152329 / part-3](https://gradle-enterprise.elastic.co/s/cenolpdgvnrxe)
- [elasticsearch-pull-request #152303 / part-3](https://gradle-enterprise.elastic.co/s/ei2f4gilied72)
- [elasticsearch-pull-request #152303 / part-3](https://gradle-enterprise.elastic.co/s/lbr5bg3nssylm)

**Reproduction Line:**
```
./gradlew ":x-pack:plugin:esql:qa:server:single-node:javaRestTest" --tests "org.elasticsearch.xpack.esql.qa.single_node.EsqlSpecForceStoredLoadingIT" -Dtests.method="test {csv-spec:rename.renameIntertwinedWithSort}" -Dtests.seed=FD8674695D5C28C9 -Dtests.locale=os -Dtests.timezone=Europe/Gibraltar -Druntime.java=26
```

**Applicable branches:**
main

**Reproduces locally?:**
N/A

**Failure History:**
[See dashboard](https://es-delivery-stats.elastic.dev/app/dashboards#/view/dcec9e60-72ac-11ee-8f39-55975ded9e63?_g=(refreshInterval:(pause:!t,value:60000),time:(from:now-7d%2Fd,to:now))&_a=(controlGroupState:(initialChildControlState:('0c0c9cb8-ccd2-45c6-9b13-96bac4abc542':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:task.keyword,order:0,selectedOptions:!(),title:'GradleTask',type:optionsListControl),'4e6ad9d6-6fdc-4fcc-bf1a-aa6ca79e0850':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:className.keyword,order:1,selectedOptions:!(org.elasticsearch.xpack.esql.qa.single_node.EsqlSpecForceStoredLoadingIT),title:'Suite',type:optionsListControl),'144933da-5c1b-4257-a969-7f43455a7901':(dataViewId:fbbdc689-be23-4b3d-8057-aa402e9ed0c5,fieldName:name.keyword,order:2,selectedOptions:!('org.elasticsearch.xpack.esql.qa.single_node.EsqlSpecForceStoredLoadingIT'),title:'Test',type:optionsListControl)))))

**Failure Message:**
```
java.lang.AssertionError: 
Data mismatch:
row 0 column 20:a list containing
row 0 column 20:0: expected "12.40" but was "14.25"
row 0 column 20:1: expected "14.25" but was "12.4"
row 2 column 20:a list containing
row 2 column 20:0: expected "-6.77" but was "8.78"
row 2 column 20:1: "0.39"
row 2 column 20:2: expected "8.30" but was "-6.77"
row 2 column 20:3: expected "8.78" but was "8.3"
row 6 column 20:a list containing
row 6 column 20:0: "-7.37"
row 6 column 20:1: "10.62"
row 6 column 20:2: expected "11.20" but was "11.2"
Actual:
avg_worked_seconds:long | birth_date:datetime      | emp_no:integer | first_name:keyword | gender:keyword | height:double | height.float:double | height.half_float:double | height.scaled_float:double | hire_date:datetime       | is_rehired:boolean          | job_positions:keyword                                                                  | languages:integer | languages.byte:integer | languages.long:long | languages.short:integer | last_name:keyword | sa
[truncated]
```

**Issue Reasons:**
- [main] 1 consecutive failures in class org.elasticsearch.xpack.esql.qa.single_node.EsqlSpecForceStoredLoadingIT
- [main] 1 consecutive failures in step part3
- [main] 1 consecutive failures in step java_rest_tests
- [main] 1 consecutive failures in pipeline elasticsearch-intake
- [main] 100 failures in class org.elasticsearch.xpack.esql.qa.single_node.EsqlSpecForceStoredLoadingIT (11.2% fail rate in 893 executions)
- [main] 8 failures in step part3 (4.6% fail rate in 175 executions)
- [main] 49 failures in step releasetestcheckesqlqa_release-tests (32.2% fail rate in 152 executions)
- [main] 36 failures in step part-3 (8.1% fail rate in 447 executions)
- [main] 5 failures in step checkpart3_release-tests (22.7% fail rate in 22 executions)
- [main] 8 failures in pipeline elasticsearch-intake (4.6% fail rate in 175 executions)
- [main] 46 failures in pipeline elasticsearch-pull-request (10.5% fail rate in 437 executions)

**Note:**
This issue was created using new test triage automation. Please report issues or feedback to es-delivery.

--- a/x-pack/plugin/esql/qa/testFixtures/src/main/resources/boolean.csv-spec
+++ b/x-pack/plugin/esql/qa/testFixtures/src/main/resources/boolean.csv-spec
@@ -208,6 +208,7 @@ row ul = [9223372036854775808, 9223372036854775807, 1, 0] | eval bool = to_bool(
 ;
 
 convertFromIntAndLong
+request_stored: skip
 from employees | sort emp_no | keep emp_no, salary_change* | eval int2bool = to_boolean(salary_change.int), long2bool = to_boolean(salary_change.long) | limit 10;
 
 emp_no:integer |salary_change:double      |salary_change.int:integer | salary_change.keyword:keyword |salary_change.long:long |int2bool:boolean          |long2bool:boolean       

--- a/x-pack/plugin/esql/qa/testFixtures/src/main/resources/drop.csv-spec
+++ b/x-pack/plugin/esql/qa/testFixtures/src/main/resources/drop.csv-spec
@@ -34,6 +34,7 @@ avg_worked_seconds:long | birth_date:date | emp_no:integer | first_name:keyword 
 
 whereWithEvalGeneratedValue_DropHeight
 skip_flattened_rewrite: field_extract_multivalue_null
+request_stored: skip
 from employees | sort emp_no | eval x = salary / 2 | where x > 37000 | drop height*;
 
 avg_worked_seconds:long | birth_date:date | emp_no:integer | first_name:keyword | gender:keyword | hire_date:date | is_rehired:boolean | job_positions:keyword | languages:integer | languages.byte:integer | languages.long:long | languages.short:integer | last_name:keyword | salary:integer | salary_change:double | salary_change.int:integer |salary_change.keyword:keyword |salary_change.long:long | still_hired:boolean | x:integer

--- a/x-pack/plugin/esql/qa/testFixtures/src/main/resources/keep.csv-spec
+++ b/x-pack/plugin/esql/qa/testFixtures/src/main/resources/keep.csv-spec
@@ -319,6 +319,7 @@ salary:integer
 
 whereWithEvalGeneratedValue
 skip_flattened_rewrite: field_extract_multivalue_null
+request_stored: skip
 // the result from running on ES is the one with many decimals the test that runs locally is the one rounded to 2 decimals
 // the "height" fields have the values as 1.7, 1.7000000476837158, 1.7001953125, 1.7
 from employees | eval x = salary / 2 | where x > 37000;

--- a/x-pack/plugin/esql/qa/testFixtures/src/main/resources/rename.csv-spec
+++ b/x-pack/plugin/esql/qa/testFixtures/src/main/resources/rename.csv-spec
@@ -161,6 +161,7 @@ y:integer | x:date
 renameIntertwinedWithSort
 skip_flattened_rewrite: field_extract_multivalue_null
 required_capability: fix_precision_of_scaled_float_fields
+request_stored: skip
 FROM employees | eval x = salary | rename x as y | rename y as x | sort x | rename x as y | limit 10;
 
 avg_worked_seconds:l | birth_date:date          | emp_no:i             | first_name:s         | gender:s             | height:d             | height.float:d       | height.half_float:d  | height.scaled_float:d| hire_date:date           | is_rehired:bool      | job_positions:s      | languages:i          | languages.byte:i     | languages.long:l     | languages.short:i    | last_name:s          | salary:i             | salary_change:d      | salary_change.int:i  | salary_change.keyword:s | salary_change.long:l | still_hired:bool  | y:i          

--- a/x-pack/plugin/esql/qa/testFixtures/src/main/resources/topN.csv-spec
+++ b/x-pack/plugin/esql/qa/testFixtures/src/main/resources/topN.csv-spec
@@ -117,6 +117,7 @@ Mary              |Sluis            |[-7.82, 3.48, 8.73, 10.35]
 
 sortingOnNumbersFromStrings
 skip_flattened_rewrite: field_extract_multivalue_null
+request_stored: skip
 from employees 
 | eval long_from_string = to_long(salary_change.keyword) 
 | sort long_from_string asc nulls last, emp_no 

